.github/dependabot.yml

```diff
@@ -99,6 +99,12 @@ updates:
       interval: weekly
     open-pull-requests-limit: 20
 
+  - package-ecosystem: nuget
```

Comment:
Do we need a follow-up issue/PR to cover Linux and mac builds of native?

Reply:
I think that we should treat native code dependencies PR created as a reminder that we should check it also for linux.

BTW based on what I see we have different versions  for Windows and Linux/MacOs.
See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/d486e2e5c590d2b5314fc326551e0758d0b61eec/src/OpenTelemetry.AutoInstrumentation.Native/CMakeLists.txt#L105
and 
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/d486e2e5c590d2b5314fc326551e0758d0b61eec/src/OpenTelemetry.AutoInstrumentation.Native/packages.config#L3